launch/check_board_position.launch

```diff
@@ -0,0 +1,9 @@
+<!-- Launching all required nodes for checking the position of the board w.r.t. the robot. -->
+<!-- Could also be used to remote controle the robot during the game. -->
+<launch>
+
+    <node name="ttt_move_maker_server" pkg="baxter_tictactoe" type="move_maker_server" args="$(find baxter_tictactoe)/resources/trajectories/trajectories_over_heap_safe.xml robot/limb/left/follow_joint_trajectory"  respawn="false" output="screen" required="true"/>
+
+    <node name="check_board_positions" pkg="baxter_tictactoe" type="move_maker_simpleclient" respawn="false" required="true" output="screen" />
+
+</launch>
```

Comment:
what does this launch file do? Is it necessary? I put the comment to remember us in sight of #8